scripts/create-agent.ts

```diff
@@ -24,10 +29,22 @@ interface TimingResult {
 class VargasJRAgentCreator {
   private ec2: EC2;
   private config: AgentConfig;
+  private readonly instanceTypeFallbacks: InstanceTypeConfig[] = [
```

Comment:
Nah instead of doing this instance fallback approach, detect the error then query for all instances that don't have an open PR in the repo. For each of them, terminate and try again. If there were none, raise an error